# Mapping Cleaned Reads Worksheet

<!--- Write name below --->
## Name:

<!--- For this worksheet, answer the following questions --->

## Q1: What does it mean to map/align reads to a reference?
Answer:

It means to take sequencing data in short chunks called reads and determining where they fit along / originate from compared to a reference genome.

## Q2: What read mapper does the mapping_cleaned_reads.sh script use?
Answer:

minimap

## Q3: Both Illumina and Nanopore reads are used for this assignment. What are the major differences between the methodology used for these sequencing platforms?
Answer:

Illumina = fluorescent nucleotides are added to a DNA template / produces short reads
Nanopore = sequences are run through small pores and each nucleotide disrupts an electric current in a unique way / produces long reads

## Q4: What differences do you notice between the Illumina and Nanopre raw_data fastq file sizes? Which are larger?
Answer:

## Q5: What differences do you notice between the Illumina and Nanopore cleaned_reads fastq file sizes? Which are larger?
Answer:

## Q6: What explains the difference in your responses of Q4 and Q5? (HINT: Take a glimpse at the raw data .fastq files themselves)
Answer:

## Q7: What is the average read depth for the Illumina data across all samples for the genomic regions that were mapped to?
Answer:

## Q8: What is the average read depth for the Illumina data across all samples for all genomic regions?
Answer:

## Q9: What is the average read depth for the Nanopore data across all samples for the genomic regions that were mapped to?
Answer:

## Q10: What is the average read depth for the Nanopore data across all samples for all genomic regions?
Answer:
